docs/v3.x/concepts/templates.md

```diff
@@ -0,0 +1,79 @@
+# Templates
+
+A template is a pre-made Strapi configuration designed for a specific use case. It allows you to quickly boostrap a custom Strapi app.
+
+Here are some things a template may configure for you:
+
+- Collection types and single types
+- Components and dynamic zones
+- Plugins to install, or custom plugins
+
+::: warning
+Templates and starters are not the same thing:
+
+- A _template_ is a pre-made Strapi configuration
+- A _starter_ is a pre-made frontend application that consumes a Strapi API
```

Comment:
frontend + backend no?

Reply:
Our starters are currently monorepos that include both the frontend and the backend, but that will change.

Once templates are released, we will convert the backend of all existing starters into new template repos. So the starters will then only have the frontend. They'll have an instruction in the readme to tell users to install the matching backend template using `create-strapi-app` and the `--template` option.

This will allow us to have one template per use case, with several matching frontends for each framework.

For example, for the portfolio use case, we may have the following repos:

* `strapi/strapi-template-portfolio`
* `strapi/strapi-starter-gatsby-portfolio`
* `strapi/strapi-starter-gridsome-portfolio`
* `strapi/strapi-starter-angular-portfolio`
* ...and many other frontend starters, probably made by community members